frontend/src/worksheets/CreateWorkSheetForm.js

```diff
@@ -45,10 +46,13 @@ function CreateWorkSheetForm({
               localDateTime: getCurrentDate(),
               workerSignature: '',
               proofOfEmployment: '',
-              status: '',
+              worksheetStatus: 'CREATED',
```

Comment:
Értem, hogy valószínűleg a backend várja így, de redundáns ez az elnevezés, mert ha worksheet objektumot kezelünk tudjuk, hogy az egy worksheet. Ennyi erővel a többi member is lehetne prefixelve: worksheetLocalDateTime, worksheetWorkerSignature stb. Láthatóan fölösleges és csak lassítja az olvasást, megértést.

Reply:
Ez igaz, de a backend így várja.

Reply:
Át lehet nevezni a backend kódban is :)

Reply:
A status már foglalt volt, ezért lett worksheetStatus.